### PR TITLE
sig-node: endocrimes as e2e_node approver

### PR DESCRIPTION
**Pull request by endocrimes**
[need a kind, but kinds don't really apply to this]
/kind cleanup
/sig node

I'm a heavy reviewer of test-infra and e2e_node changes and member of the CI testing subgroup, applying for test-infra approver too: https://github.com/kubernetes/test-infra/pull/26000

#### Special notes for your reviewer:

Reviewer since 18 Nov 2021: https://github.com/kubernetes/kubernetes/pull/106493
Reviewed merged PRs https://github.com/kubernetes/kubernetes/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Aendocrimes+label%3Aarea%2Ftest

sponsored by: @ehashman 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by ehashman** [comment]
+1 (binding)

But I repeat myself: https://github.com/kubernetes/test-infra/pull/26000#issuecomment-1102826101

**Comment by dims** [comment]
+1 from me!

**Comment by endocrimes** [comment]
/cc @dchen1107 @derekwaynecarr @SergeyKanzhelev 

(as approvers of the other PR, Sig Leads, and CI Subproject co-lead)

**Review by odinuge — APPROVED** [review]
+1

Thanks for all your work @endocrimes!

**Review by SergeyKanzhelev — APPROVED** [review]
/hold

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/priority important-soon
/triage accepted

ok, I think I corrected all labels now, waiting for code thaw

**Comment by aojea** [comment]
+1

**Pull request merged**
